lib/metasploit/framework/data_service/remote/http/remote_host_data_service.rb

```diff
@@ -32,4 +33,20 @@ def update_host(opts)
   def delete_host(opts)
     json_to_mdm_object(self.delete_data(HOST_API_PATH, opts), HOST_MDM_CLASS)
   end
-end
\ No newline at end of file
+
+  def get_host_tags(opts)
+    path = get_path_select(opts, HOST_API_PATH) + '/tag'
```

Comment:
since the `hosts` path is plural it probably makes sense for the `tag` path to be plural so `tags` aswell

Reply:
👍